### PR TITLE
Create Kubernetes YAML for Couchbase Server

### PR DESCRIPTION
**Pull request by yudaykiran**
**What this PR does / why we need it**:
- Contains YAML for creating a StatefulSet
- Contains YAML for creating a Headless Service and NodePort Service.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #
Fix for #709 

**Special notes for your reviewer**:

Signed-off-by: yudaykiran <udaykiran.y@gmail.com>

### CODE REVIEW

File: k8s/demo/couchbase/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+# Couchbase
+
+This document demonstrates the deployment of Counchbase as a StatefulSet in a Kubernetes Cluster. The user can spawn a Couchbase StatefulSet that will use OpenEBS as its persistent storage.
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: Counchbase = Couchbase
Kubernetes Cluster = cluster

File: k8s/demo/couchbase/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+## Deploy as a StatefulSet
+
+Deploying Couchbase as a StatefulSet provides the following benefits:
+
+- Stable, unique network identifiers.
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: remove ,

File: k8s/demo/couchbase/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+- Stable persistent storage.
+- Ordered, graceful deployment and scaling.
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: Ordered graceful deployment is one word. If so remove ,

File: k8s/demo/couchbase/README.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+- Ordered, graceful deletion and termination.
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: same comment as above

File: k8s/demo/couchbase/README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+```bash
+ubuntu@kubemaster:~kubectl get nodes
+NAME            STATUS    AGE       VERSION
+kubemaster      Ready     3d        v1.8.2
+kubeminion-01   Ready     3d        v1.8.2
+kubeminion-02   Ready     3d        v1.8.2
+
+```
+
+Download and apply the Couchbase YAML from OpenEBS repo:
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: repo=repository

File: k8s/demo/couchbase/README.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+couchbase                                           ClusterIP   None             <none>        8091/TCP            11h
+couchbase-ui                                        NodePort    10.103.161.153   <none>        8091:30438/TCP      11h
+kubernetes                                          ClusterIP   10.96.0.1        <none>        443/TCP             3d
+maya-apiserver-service                              ClusterIP   10.111.26.252    <none>        5656/TCP            3d
+
+```
+
+## Launch Couchbase Server
+
+The Couchbase service YAML, creates a service of type NodePort to make the Couchbase Server available outside the cluster.
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: creates a service of type = creates a NodePort service type
Server = server
to make the Couchbase Server = for making the Couchbase server

File: k8s/demo/couchbase/README.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+
+Get the IP Address of the node running the Couchbase Server:
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: Get the IP Address of the node = Get the node's IP Address which is running
Server = server

File: k8s/demo/couchbase/README.md [hunk 8]
```diff
@@ -0,0 +1,148 @@
...
+```
+Get the port number from the Couchbase UI service:
+
+```bash
+ubuntu@kubemaster:~ kubectl describe svc couchbase-ui | grep NodePort:
+NodePort:		couchbase  30438/TCP
+
+```
+
+Open the below URL in the browser:
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: below = following

File: k8s/demo/couchbase/README.md [hunk 9]
```diff
@@ -0,0 +1,148 @@
...
+
+The default Username is Administrator and Password is password. Enter the credentials to see the console.
+
+![Couchbase Cluster]
+
+__Click on Server Nodes to see how many Couchbase nodes are part of the cluster. As expected, it shows only one node__
+
+![Couchbase Server]
+
+__Click on Data Buckets to see a sample bucket that was created as part of the image__
```
Comment (mahebbar) [inline k8s/demo/couchbase/README.md]: Click on Data Buckets = Click Data Buckets

**Comment by yudaykiran** [comment]
Changes Implemented.

**Pull request closed without merging**